lib/nanoc/extra/checking/checks/mixed_content.rb

```diff
@@ -0,0 +1,31 @@
+# encoding: utf-8
+
+PROTOCOL_PATTERN = /^(\w+):\/\//
```

Comment:
Could you move this into the `MixedContent` class? This way, `PROTOCOL_PATTERN` won’t be a top-level constant. Constant lookup will still work.